### PR TITLE
maint window and upgrade

### PR DESCRIPTION
**Pull request by keichwa**
per feedback, see
https://github.com/SUSE/doc-susemanager/pull/781#discussion_r332868061

### CODE REVIEW

File: modules/administration/pages/maintenance-window.adoc [hunk 1]
```diff
@@ -53,8 +53,10 @@ complete procedure, also see above:
 By default, several update channels are configured and enabled for the {susemgr} Server.
 New and updated packages will become available automatically.
 
+ifeval::[{suma-content} == true]
 To keep {susemgr} up to date, either connect it directly to {scc} or use {rmtool} (RMT).
 You can use RMT as a local installation source for disconnected environments.
+endif::[]
 
 You can check that the update channels are available on your system with this command:
```
Comment (juliogonzalez) [inline modules/administration/pages/maintenance-window.adoc]: Unrelated to this PR?

The content below only applies to SUSE Manager, not Uyuni. As we neither have MUs, I'd say you could move the `endif::[]` so it applies until line 99.

File: modules/administration/pages/maintenance-window.adoc [hunk 2]
```diff
@@ -53,8 +53,10 @@ complete procedure, also see above:
 By default, several update channels are configured and enabled for the {susemgr} Server.
```
Comment (juliogonzalez) [inline modules/administration/pages/maintenance-window.adoc]: Un related to this PR, shouldn't we use `{productname}` above?

File: modules/administration/pages/maintenance-window.adoc (hunk 1 above)
Comment (keichwa) [reply modules/administration/pages/maintenance-window.adoc]: ok, but ifdefs must not exceed more than 1 page (~20 lines) ;)  That's my very personal rule. I either will create an include file or create 1 file file for suma and the other one for uyuni (and put the ifdef in the upper-level nav-file).

File: modules/administration/pages/maintenance-window.adoc (hunk 2 above)
Comment (keichwa) [reply modules/administration/pages/maintenance-window.adoc]: probably yes.  We sometimes to drive-by edits ;)

File: modules/administration/pages/maintenance-window.adoc (hunk 1 above)
Comment (keichwa) [reply modules/administration/pages/maintenance-window.adoc]: done

File: modules/administration/pages/maintenance-window.adoc (hunk 2 above)
Comment (keichwa) [reply modules/administration/pages/maintenance-window.adoc]: done

File: modules/upgrade/pages/db-migration.adoc [hunk 3]
```diff
@@ -4,10 +4,10 @@
 This is about PostgreSQL database migration.
 
 If you want to upgrade {productname} 3.2 to {productname} 4 you must check whether {productname} 3.2 is already based on PostgreSQL version 10.
```
Comment (juliogonzalez) [inline modules/upgrade/pages/db-migration.adoc]: Not for this PR: I'd rewrite this doc so it fits both SUSE Manager or Uyuni as:

```
If you want to upgrade to the latest {productname} version you must check whether {productname} Server is already based on PostgreSQL version 10.
```

And then below, remove all mentions to `3.2` except where it's REALLY needed (and in that case enclose that with a diff).

**Pull request merged**